R/writer.R

```diff
@@ -153,7 +154,9 @@ write_masked_functions <- function(){
 write_used_functions <- function(){
    used_functions_list <- get_log_element("used_packages_functions")
 
-   combined <- aggregate(function_name~library, used_functions_list, paste)
+   combined <- used_functions_list %>%
+      group_by(library) %>%
```

Comment:
@nicholas-masel I'm not sure what's downstream but does this dataframe need to be ungrouped for any reason down the line?  Not sure if an `ungroup()` is needed after the `summarize()` or not

Reply:
@kodesiba Good comment and I checked into the documentation to be sure.  Since I am using `summarize()` after `group()` and I only have one group, the resulting df is ungrouped as the last grouping is dropped by default.

From `summarize()`:
> When .groups is not specified, it is chosen based on the number of rows of the results:
> If all the results have 1 row, you get "drop_last".

> .groups - Grouping structure of the result.
> "drop_last": dropping the last level of grouping. This was the only supported option before version 1.0.0.